src/util/source_location.h

```diff
@@ -162,6 +162,10 @@ class source_locationt:public irept
     return is_built_in(id2string(get_file()));
   }
 
+  /// Set all unset source-location fields in 'to' to their values in
+  /// 'from'. Leave set fields in 'to' alone.
```

Comment:
Here, there's neither `from` nor `to`, this would make more sense at the implementation site.